docs/source/command_line_tools/vr_run.md

```diff
@@ -0,0 +1,25 @@
+# Command line tools overview
+
+The `virtual_rainforest` package currently provides a single command line tool `vr_run`.
+This is tool is likely to be superseded as the project develops, either by a set of
+command line tools or a single more generic tool.
+
+## The `vr_run` command line tool
+
+The `vr_run` command line tool exists to run the virtual rainforest model. At the moment
+this consists of validated a set of schema files, using their contents to initialise a
+set of models and then setting up the main simulation loop. It goes no further than
+that, and no actual simulation steps are performed. This will change in the future.
+
+### Using `vr_run`
+
+The usage instructions for `vr_run` are given below:
+
+```{include} command_line_usage/vr_run.txt
+```
```

Comment:
I don't think this is needed. Users can just run `vr_run --help` themselves and see the output. You can tell them to run this for more info if you like, but I don't think you need to go to the extent of actually including the output in the documentation.

While we're on the topic, what happens if you run `vr_run` without arguments? Or with bad arguments? In those cases it would also be worth pointing users towards `vr_run --help`.

Reply:
Good point about running `vr_run` without arguments (or with bad arguments) at the moment that just raises a critical error without telling the user where the problem can be resolved. Which now I think about isn't very useful for anyone who doesn't already know how the package works

Reply:
Ok cool. I think it's totally fine to leave an error message in there btw, but I'd just print another line saying "For more information please run vr_run --help" (or something)

Reply:
One of the benefits of including the command output here is that it gives us somewhere in the docs to check that the command line output is formatted properly, which is hard to work out from the docstring alone. With normal functions their docstrings are displayed in the api, but as far as I'm aware it doesn't make sense to add in command line entry points there?

Reply:
I actually disagree relatively strongly. It does no real _harm_ here and it does provide a prospective user with the ability to read and learn about the interface without _having_ to install it. If I was teaching a student to use this tool, having a page where they could review the main interfaces would be a useful step. This is partly a _tutorial_ page, so hand-holding is completely in its place here.

Reply:
Maybe. If you want to explain what the different options mean here, then I think that's fine. I mostly just object to having a script to dump the program output into your documentation for the purposes of checking that it's formatted correctly!

Up to you though.